tdrs-backend/temp_debug_script.py

```diff
@@ -0,0 +1,13 @@
+import logging
```

Comment:
I think for a formal PR we should delete our debugger script since we determined this was not at cause.